tox/venv.py

```diff
@@ -178,7 +186,7 @@ def update(self, action):
                 self.envconfig.deps, v)
 
     def _getliveconfig(self):
-        python = self.envconfig.python_info.executable
+        python = str(self.envconfig.python_info.executable)
```

Comment:
Not sure why this was necessary, but I cannot reproduce the test failure that had been fixed during development of this comment. Thanks for pointing this out!